ui/src/components/ui/alert.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react'
+import { cva, type VariantProps } from 'class-variance-authority'
+
+import { cn } from '@/components/ui/utils'
+
+const alertVariants = cva(
+    // eslint-disable-next-line @stylistic/max-len
+    'relative w-full rounded-lg border border-slate-200 p-4 [&>svg~*]:pl-7 [&>svg+div]:translate-y-[-3px] [&>svg]:absolute [&>svg]:left-4 [&>svg]:top-4 [&>svg]:text-slate-950 dark:border-slate-800 dark:[&>svg]:text-slate-50',
+    {
+        variants: {
+            variant: {
+                default: 'bg-white text-slate-950 dark:bg-slate-950 dark:text-slate-50',
+                destructive:
+                // eslint-disable-next-line @stylistic/max-len
+          'border-red-500/50 text-red-500 dark:border-red-500 [&>svg]:text-red-500 dark:border-red-900/50 dark:text-red-900 dark:dark:border-red-900 dark:[&>svg]:text-red-900',
```

Comment:
For the eslint disable lines, I got around this by using the back-tick `` instead of ''. That way I can put the text on the next line. You can take a look at `button.tsx` as an example.